src/lib/seed.ts

```diff
@@ -131,10 +131,10 @@ export class Seed {
     projects: { count: number };
     users: { count: number };
   }> {
-    return new Request("/stats", this.api).get();
+    return new Request("/stats", this.addr).get();
   }
 
-  static async getPeer(host: Host): Promise<{ id: string }> {
+  static async getNode(host: Host): Promise<{ id: string }> {
```

Comment:
why not create both `getNode` and `getPeer`, also with comments (that one is for heartwood, etc)?

If you want to keep the heartwood fn signature but call `/peer`, how are you planning to eventually resolve this?

Reply:
That's for a follow up. I'd like to first just extract the renames out of #515.

Reply:
hm... I still suggest a comment explaining the discrepancy, this might be helpful for your future self, but if you're certain then this lgtm as well.